src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -1631,6 +1665,29 @@ components:
         - electoralRollNumber
         - deliveryAddressType
 
+    UpdateAnonymousElectorDocumentRequest:
+      title: UpdateAnonymousElectorDocumentRequest
+      description: Request body to update specific details on one or more Anonymous Elector Documents (AEDs). Note that explicitly setting any of the values to an empty string ("") indicates it should be removed. Otherwise, omitting the field means it will not be updated.
+      type: object
+      properties:
+        sourceReference:
+          type: string
+          example: 1f0f76a9a66f438b9bb33070
+          description: Reference in the source application that this Anonymous Elector Document is for.
+        email:
+          type: string
+          maxLength: 1024
+          format: email
+          description: The applicant's email address. This will only be updated if a non-null value is provided. Note that it will be removed if an empty string ("") is sent (if applicable).
+          example: fred.blogs@some-domain.co.uk
+        phoneNumber:
+          type: string
+          maxLength: 50
+          description: The applicant's phone number. This will only be updated if a non-null value is provided. Note that it will be removed if an empty string ("") is sent (if applicable).
+          example: 01234 567890
+      required:
+        - sourceReference
```

Comment:
What happens when a call to this endpoint is made without email and phone in body? Are they both set to blank/empty. Don't think that should be allowed

Reply:
The current plan is that nothing will happen (you explicitly have to set them to `""` to remove them, but I need to check that use case). 

Alternatively, if nothing is sent in the request, then we could return a 400. That might be better actually 🤔

Reply:
Yeah throwing `400` error would be better when both contact details values are not present in body.

Reply:
I plan to do that, but I haven't mentioned it in the spec yet. I'm going to wait until we have consensus re the applicationId in the path vs body.